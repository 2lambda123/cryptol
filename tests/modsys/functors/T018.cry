module T1 where

parameter
  anything : Bit

submodule A where
  x = 0x02

import submodule A

y = x
