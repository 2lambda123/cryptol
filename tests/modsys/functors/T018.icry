:load T018.cry
y
