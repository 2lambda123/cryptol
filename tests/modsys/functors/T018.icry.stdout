Loading module Cryptol
Loading module Cryptol
Loading interface module `parameter` interface of T1
Loading module T1

Expression depends on definitions from a parameterized module:
  T1::y
